spec/services/metrc_service/base_spec.rb

```diff
@@ -258,43 +258,49 @@
   end
 
   context '#get_resource_unit' do
+    include_context 'with synced data' do
+      let(:facility_id) { 1 }
+      let(:batch_id) { 331 }
+    end
+
     before do
-      stub_request(:get, 'https://portal.artemisag.com/api/v3/facilities/1568')
-        .to_return(body: { data: { id: '1568', type: 'facilities', attributes: { id: 1568, name: 'Rare Dankness' } } }.to_json)
+      stub_request(:get, "https://portal.artemisag.com/api/v3/facilities/#{facility_id}")
+        .to_return(body: load_response_json("api/sync/facilities/#{facility_id}"))
```

Comment:
Can you please move the `load_response_json`. and maybe rename it to `load_stubbed_json`, to a spec helper method?

Reply:
nvm